packages/kyodo-contracts/contracts/MembersV2.sol

```diff
@@ -10,12 +10,23 @@ contract MembersV2 is MembersV1 {
     external
   {
     require(nickNameNotExist(_value));
+    
+    bool isAliasEmpty = _value.toSlice().empty();
+
     string storage prevValue = whitelist[msg.sender].alias;
+
+    if (isAliasEmpty) {
+      require(!prevValue.toSlice().empty());
```

Comment:
Shall we start writing an error message as a second param in `require` function?

Reply:
Added in the latest commit